neo/tools/compilers/roqvq/roq.m

```diff
@@ -49,7 +49,7 @@ - (BOOL)isLastFrame
 	return lastFrame;
 }
 
-- (BOOL)scaleable
+- (BOOL)scalable
```

Comment:
no idea what this objective C file is really used for, but I'm not a fan of fixing typos in actual code.

(comments and error message strings should be ok, everything else seems risky)